edit-post/components/header/pinned-plugins/style.scss

```diff
@@ -4,4 +4,24 @@
 	.components-icon-button {
 		margin-left: 4px;
 	}
+
+	// Colorize plugin icons to ensure contrast and cohesion, but allow plugin devs to override.
```

Comment:
This is repeated, which seems strange. But if it's staying we should at least say "developers".